app/workers/spaces/bust_caches_for_space_change_worker.rb

```diff
@@ -0,0 +1,16 @@
+module Spaces
+  # Conditionally delete the async cache when a Space changes
+  #
+  # @note Good news for those of you using Redis as your cache store, you have access to `Rails.cache.delete_matched`
+  #
+  # @see https://api.rubyonrails.org/classes/ActiveSupport/Cache/RedisCacheStore.html#method-i-delete_matched
+  class BustCachesForSpaceChangeWorker < BustCacheBaseWorker
```

Comment:
nit: unless this class is meant to do more I would prefer to hone in on clarification `BustTagSidebarCachesWorker`

Reply:
It's possible we'll need more…our user acceptance test is finding a few things.